systemtests/src/main/java/io/enmasse/systemtest/utils/TestUtils.java

```diff
@@ -563,4 +570,37 @@ public static void waitForChangedResourceVersion(final TimeoutBudget budget, fin
             }
         }, budget);
     }
+
+    public static void waitForAddressSpaceReady(final String namespace, final String name, final TimeoutBudget budget) throws Exception {
```

Comment:
The methods added to this class are never used, delete it please

Reply:
I'll do that, plus other minor changes

Reply:
@famartinrh please wait … I just pushed a new commit that makes use of it

Reply:
@ctron there is already a method for checking if address space is ready `io.enmasse.systemtest.utils.AddressSpaceUtils#waitForAddressSpaceReady`

Reply:
I didn't know that :smiley: … pushed a new version that re-uses that.